blobstore-findbugs-exclude-filter.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
7.2.0 oss parent not exclude SE_BAD_FIELD bug pattern (Class org.everit.blobstore.jdbc.schema.qdsl.QBlobstoreBlob defines non-transient non-serializable instance field pk).